server/src/routes/api/v1/tripVotesRouter.js

```diff
@@ -0,0 +1,59 @@
+import express from "express"
+
+import Vote from "../../../models/Vote.js"
+
+const tripVotesRouter = new express.Router({ mergeParams: true })
+
+tripVotesRouter.get("/", async (req, res) => {
+  const userId = req.user.id
+  const { tripId } = req.params
+
+  try {
+    const votes = await Vote.query().where({ tripId: tripId })
+    const userVote = votes.find((element) => element.userId === userId)
```

Comment:
Wondering why you're calculating the total average of votes the user made for a given trip? We would want to display the average total of _all_ votes made for a trip by all users